addons/istio/istio.yaml

```
@@ -6,17 +6,9 @@ metadata:
   labels:
     kubeaddons.mesosphere.io/name: istio
   annotations:
-    catalog.kubeaddons.mesosphere.io/addon-revision: "1.6.12-1"
-    appversion.kubeaddons.mesosphere.io/istio: "1.6.12"
-    appversion.kubeaddons.mesosphere.io/kiali: "1.18.0"
-    appversion.kubeaddons.mesosphere.io/jaeger: "1.16.0"
-    stage.kubeaddons.mesosphere.io/kiali: Preview
-    stage.kubeaddons.mesosphere.io/jaeger: Preview
-    endpoint.kubeaddons.mesosphere.io/kiali: "/ops/portal/kiali"
-    endpoint.kubeaddons.mesosphere.io/jaeger: "/ops/portal/jaeger"
+    catalog.kubeaddons.mesosphere.io/addon-revision: "1.8.1-1"
+    appversion.kubeaddons.mesosphere.io/istio: "1.8.1"
     docs.kubeaddons.mesosphere.io/istio: "https://istio.io/docs/"
-    docs.kubeaddons.mesosphere.io/kiali: "https://istio.io/docs/tasks/telemetry/kiali/"
-    docs.kubeaddons.mesosphere.io/jaeger: "https://istio.io/docs/tasks/telemetry/distributed-tracing/jaeger/"
     values.chart.helm.kubeaddons.mesosphere.io/istio: "https://raw.githubusercontent.com/mesosphere/charts/a7d3c02bac8c4c3ae02854e60483b97cfb80b1b7/staging/istio/values.yaml"
```

```suggestion
    values.chart.helm.kubeaddons.mesosphere.io/istio: "https://raw.githubusercontent.com/mesosphere/charts/1079b3c/staging/istio/values.yaml"
```
just a values change grabbed from #775